package.json

```diff
@@ -40,12 +40,12 @@
     },
     "dependencies": {
         "@azure/functions": "^1.2.3",
-        "axios": "^0.21.1",
+        "axios": "^1.4.0",

```

Comment:
Latest one is ^1.6.0. Version 1.4.0 doesn't even contains security fix. 
Also looking at package-lock.json it seems like dependency was updated incorrectly